src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,16 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div>
+    {comments.length ? (
+      comments.map(comment => (
+        <div className="CommentList">
```

Comment:
Class CommentList is being assigned to each comment. It should be assigned to the parent div only.